src/main/java/com/exadel/discountwebapp/history/mapper/HistoryMapper.java

```diff
@@ -0,0 +1,36 @@
+package com.exadel.discountwebapp.history.mapper;
+
+import com.exadel.discountwebapp.history.vo.HistoryRequestVO;
+import com.exadel.discountwebapp.history.vo.HistoryResponseVO;
+import org.javers.core.Changes;
+import org.javers.core.diff.changetype.ValueChange;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class HistoryMapper {
+    private final HistoryResponseVO historyResponseVO = new HistoryResponseVO();
+
+    public List<HistoryResponseVO> toVO(Changes changes, HistoryRequestVO historyRequestVO) {
+
+        ArrayList<HistoryResponseVO> build = new ArrayList<>();
```

Comment:
here let's use abstract type instead of specific
List<HistoryResponseVO> build = new ArrayList<>()

Reply:
used. but IDEA show a hint, that to use raw type is not flavourible